data/archived_mur_citation_map.csv

```diff
@@ -0,0 +1,252 @@
+2:431,52:30101
```

Comment:
1. We should eliminate the duplicated lines
2. We should not name the file with an _archived_ prefix
3. Better still, why not eliminate the CSV file, reading and caching it and have a map in Python like the old `reclassify_statutory_citation`?